blog/project-portal-completed.md

```diff
@@ -0,0 +1,140 @@
+
+## CCV – from Prototype to Production
+
+CCV started work on the Project Portals in January 2022. [Mary McGrath](https://github.com/mcmcgrath13), [Joshua Lu](https://ccv.brown.edu/people/Joshua%20Lu), [John Holland](https://ccv.brown.edu/people/John%20Gerrard%20Holland), [Aisulu Omar](https://github.com/AisOmar), [Heather Yu](https://ccv.brown.edu/people/Heather%20Yu), [Jordan Lawson](https://ccv.brown.edu/people/Jordan%20Lawson) and [Joselynn Wallace](https://ccv.brown.edu/people/Joselynn%20Wallace) were involved over the course of our two-year engagement. 
+
+Our role was to help develop the website further:
+* Build a [Project Portal for the Government of North Carolina](https://projectportal.nc.gov),
+* Convert the prototype into a JavaScript package which could easily scale to ten sites for different local governments, with support for their individual requirements, 
+* Add a Content Management System (CMS) so the site could be updated directly by governmental staff, and not just the staff of The Policy Lab, and
+* Add new features, like search, images and web analytics.
+
+The prototype was set up as a single repository containing the data for both the San Antonio and North Carolina portals, each in its own directory. We used different commands: `npm build:satx` and `npm build:nc` commands to build each version of the site. 
+
+The benefits were compelling:
+* **All the code in one place**: A single repository makes it trivial to download and install the code for any and all of the sites. Deploying all the sites needed only a single interaction – triggering the GitHub Actions deployment script on the single repo.
+* **Low-effort changes**: Any changes made on one site would automatically be reflected on all the others.
+
+The drawbacks were major:
+* **Tight coupling**: By having all the code in a single repository and having all the sites deploy immediately whenever we merged code into the [main branch](https://www.atlassian.com/continuous-delivery/continuous-integration/trunk-based-development), we risked outages if, inevitably, a faulty bit of code made it through our pull request review process.
+* **Separation of concerns**: The single repository would get more and more unwieldy as new jurisdictions were added, and if one of them had totally different requirements (like having to be hosted on a different service) then we would have to fork the repository and maintain duplicate codebases. 
```

Comment:
Just a recommendation, would something like `Lack of modularization for scalability and customization` better capture the need to split the single repo into multiple repos?

Reply:
Addressing `Separation of concerns`, sorry it's not clearly referenced

Reply:
That's a really good point. How about I summarize it as "limited scalability"? That feels to me like the fundamental thing here.

Reply:
Love it!